CHANGELOG.md

```diff
@@ -2,9 +2,16 @@
 
 Full documentation for RPP is available at [https://rocm.docs.amd.com/projects/rpp/en/latest](https://rocm.docs.amd.com/projects/rpp/en/latest)
 
+## (Unreleased) RPP 1.9.7
+
+### Changes
+
+* RPP Threshold on HOST and HIP
+
 ## (Unreleased) RPP 1.9.4
 
 ### Changes
+
```

Comment:
Is 1.9.4 unreleased?

Reply:
I think this is not released yet. It will go in 6.4